client/src/Molecules.elm

```diff
@@ -390,23 +390,28 @@ vectorDecoder =
 -- Encoders
 
 
-vectorEncoder : Vector -> Encode.Value
+vectorEncoder : Maybe Vector -> Encode.Value
 vectorEncoder vector =
     case vector of
-        Level0Vec vec ->
+        Just (Level0Vec vec) ->
             level0Encoder vec
 
-        BackboneVec vec ->
+        Just (BackboneVec vec) ->
             backboneEncoder vec
 
-        LevelNVec vec ->
+        Just (LevelNVec vec) ->
             levelNEncoder vec
 
+        _ ->
+            backboneEncoder initBackbone
+
 
 backboneEncoder : Backbone -> Encode.Value
 backboneEncoder backbone =
     Encode.object
-        [ ( "name", Encode.string backbone.name )
+        [ ( "id", Encode.int backbone.id )
+        , ( "sequence_length", Encode.int backbone.sequenceLength )
+        , ( "name", Encode.string backbone.name )
```

Comment:
I don't think this is correct. See `server/app/schemas.py:VectorIn`. No `id`, no `sequence_length`.

Reply:
Indeed, we don't need them for submitting vectors